src/components/PeriodDimension/__tests__/fixedPeriods.spec.js

```diff
@@ -14,23 +16,23 @@ describe('fixedPeriods utils', () => {
             const periodIds = getFixedPeriodsOptions().map(option => option.id)
 
             expect(periodIds).toEqual([
-                'Daily',
-                'Weekly',
-                'Bi-weekly',
-                'Weekly (Start Wednesday)',
-                'Weekly (Start Thursday)',
-                'Weekly (Start Saturday)',
-                'Weekly (Start Sunday)',
-                'Monthly',
-                'Bi-monthly',
-                'Quarterly',
-                'Six-monthly',
-                'Six-monthly April',
-                'Yearly',
-                'Financial year (Start November)',
-                'Financial year (Start October)',
-                'Financial year (Start July)',
-                'Financial year (Start April)',
+                'DAILY',
+                'WEEKLY',
+                'BIWEEKLY',
+                'WEEKLYWED',
+                'WEEKLYTHU',
+                'WEEKLYSAT',
+                'WEEKLYSUN',
+                'MONTHLY',
+                'BIMONTHLY',
+                'QUARTERLY',
+                'SIXMONTHLY',
+                'SIXMONTHLYAPR',
+                'YEARLY',
+                'FYNOV',
+                'FYOCT',
+                'FYJUL',
+                'FYAPR',
```

Comment:
Here's the part where I changed the period-type IDs as I mentioned in the PR description.

Reply:
I'm guessing you don't object to these changes, so will resolve this conversation

Reply:
@janhenrikoverland @martinkrulltott @edoardo just to confirm... this doesn't ever get sent to the server right?  Shouldn't we match the server-defined period IDs regardless?

Reply:
I had the same reflection, but as I understand it these IDs are only used to control which period _**type**_ to display and has nothing to do with the periods themselves. Highlighted in green below. This is only used for filtering and shouldn't be passed to the server.
![image](https://user-images.githubusercontent.com/12590483/86444468-0db0b680-bd11-11ea-809d-2917a39c6c31.png)

Reply:
> Shouldn't we match the server-defined period IDs regardless?

I've had a quick search in the dhis2-core codebase. When you mention server-defined period IDs, do you mean these enums?
https://github.com/dhis2/dhis2-core/blob/e1c12681de3f9a7b28ee29865f7afd25ad3e8f83/dhis-2/dhis-api/src/main/java/org/hisp/dhis/calendar/DateUnitType.java#L38-L55

Reply:
Discussed this with @martinkrulltott on Slack and we can keep the current IDs. Resolving this conversation now.